index.html

```diff
@@ -84,6 +84,263 @@ <h1 class="primary-headline-text">Hey there. I’m Ansderly. <br>
+                <div class="infos-container">
+                    <div class="name-single-work">
+                        <h1>Multi-Post Stories Gain+Glory</h1>
+                    </div>
+                    <div class="stack-single-work-container">
+                        <ul class="stack-single-container">
+                            <li class="stack"><a href="#">Ruby on rail</a></li>
+                            <li class="stack"><a href="#">css</a></li>
+                            <li class="stack"><a href="#">Javascript</a></li>
+                            <li class="stack"><a href="#">html</a></li>
+                        </ul>
+                    </div>
+                    <div class="see-project-container">
+                        <div class="see-project-button">
+                            <a class="see-project-text" href="#">See Project</a>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="single-card-container">
+            <div class="single-card">
+                <div class="img-single-work">
+                </div>
+                <div class="infos-container">
+                    <div class="name-single-work">
+                        <h1>Multi-Post Stories Gain+Glory</h1>
+                    </div>
+                    <div class="stack-single-work-container">
+                        <ul class="stack-single-container">
+                            <li class="stack"><a href="#">Ruby on rail</a></li>
+                            <li class="stack"><a href="#">css</a></li>
+                            <li class="stack"><a href="#">Javascript</a></li>
+                            <li class="stack"><a href="#">html</a></li>
+                        </ul>
+                    </div>
+                    <div class="see-project-container">
+                        <div class="see-project-button">
+                            <a class="see-project-text" href="#">See Project</a>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+    </section>
+
+    <section class="about-me">
+        <div class="about-me-top">
+            <div class="about-me-text-container">
+                <div class="about-me-img-top">
+                </div>
+                <h1 class="about-me-text">About Me</h1>
+
+                <p class="about-me-description">
+                    I can help you build a product , feature or website Look through some of my work and experience!
+                    If
+                    you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.
+                </p>
+            </div>
+            <div class="get-my-resume-container">
+                <div class="get-my-resume-button">
+                    <p class="get-my-resume-text">Get My Resume</p>
+                </div>
+            </div>
```

Comment:
- [ ] Good job on the design, But here it would be wise to use a `button` or an `a` tag as this is going to be a link to your resume in the future.